### PR TITLE
Patch/globalfee multipliers

### PR DESCRIPTION
**Pull request by edjroz**

### CODE REVIEW

File: x/globalfee/ante/fee.go [hunk 1]
```diff
@@ -164,21 +174,22 @@ func (mfd FeeDecorator) GetMaxTotalBypassMinFeeMsgGasUsage(ctx sdk.Context) (res
...
 
-	requiredFees := make(sdk.Coins, len(minGasPrices))
-	// Determine the required fees by multiplying each required minimum gas
-	// price by the gas limit, where fee = ceil(minGasPrice * gasLimit).
-	glDec := sdk.NewDec(gasLimit)
-	for i, gp := range minGasPrices {
-		fee := gp.Amount.Mul(glDec)
-		requiredFees[i] = sdk.NewCoin(gp.Denom, fee.Ceil().RoundInt())
-	}
+	ctx.Logger().Error("[DEBUG]:", "minGasPrices  is ", minGasPrices)
```
Comment (ash-burnt) [inline x/globalfee/ante/fee.go:184]: Should probably change this to a ctx.Logger().Debug() if you want to leave it in

File: integration_tests/minimum_fee_test.go [hunk 2]
```diff
@@ -23,9 +24,11 @@ func TestXionMinimumFeeDefault(t *testing.T) {
 	}
 
 	t.Parallel()
-	td := BuildXionChain(t, "0.0uxion", ModifyInterChainGenesis(ModifyInterChainGenesisFn{ModifyGenesisShortProposals}, [][]string{{votingPeriod, maxDepositPeriod}, {defaultMinGasPrices.String()}}))
+	td := BuildXionChain(t, "0.1uxion", ModifyInterChainGenesis(ModifyInterChainGenesisFn{ModifyGenesisShortProposals}, [][]string{{votingPeriod, maxDepositPeriod}, {defaultMinGasPrices.String()}}))
 
 	assertion := func(t *testing.T, ctx context.Context, xion *cosmos.CosmosChain, xionUser ibc.Wallet, recipientAddress string, fundAmount int64) {
+		fmt.Println("waiting check logs")
+		time.Sleep(30 * time.Second)
```
Comment (ash-burnt) [inline integration_tests/minimum_fee_test.go:31]: Do we want this wait all the time?

File: integration_tests/minimum_fee_test.go [hunk 3]
```diff
@@ -35,6 +38,9 @@ func TestXionMinimumFeeDefault(t *testing.T) {
 		)
 		require.NoError(t, err)
 
+		fmt.Println("waiting check logs")
+		time.Sleep(30 * time.Second)
```
Comment (ash-burnt) [inline integration_tests/minimum_fee_test.go:42]: See above

File: x/globalfee/ante/fee.go [hunk 4]
```diff
@@ -85,14 +91,17 @@ func (mfd FeeDecorator) GetTxFeeRequired(ctx sdk.Context, tx sdk.FeeTx) (sdk.Coi
 
 	// Get local minimum-gas-prices
 	localFees := GetMinGasPrice(ctx, int64(tx.GetGas()))
-	return CombinedFeeRequirement(globalFees, localFees)
+	ctx.Logger().Error("[DEBUG]:", "local fee is ", localFees)
```
Comment (ash-burnt) [inline x/globalfee/ante/fee.go:94]: These should probably be debugs as well

File: x/globalfee/ante/fee.go [hunk 5]
```diff
@@ -105,19 +114,20 @@ func (mfd FeeDecorator) GetGlobalFee(ctx sdk.Context, feeTx sdk.FeeTx) (sdk.Coin
...
 	}
-	requiredGlobalFees := make(sdk.Coins, len(globalMinGasPrices))
-	// Determine the required fees by multiplying each required minimum gas
-	// price by the gas limit, where fee = ceil(minGasPrice * gasLimit).
-	glDec := sdk.NewDec(int64(feeTx.GetGas()))
-	for i, gp := range globalMinGasPrices {
-		fee := gp.Amount.Mul(glDec)
-		requiredGlobalFees[i] = sdk.NewCoin(gp.Denom, fee.Ceil().RoundInt())
-	}
+	ctx.Logger().Error("[DEBUG]:", "global fee is ", globalMinGasPrices)
```
Comment (ash-burnt) [inline x/globalfee/ante/fee.go:120]: Same

**Review by ash-burnt — CHANGES_REQUESTED** [review]

File: integration_tests/minimum_fee_test.go (hunk 2 above)
Comment (edjroz) [reply integration_tests/minimum_fee_test.go]: No, this was leftover from debugging. removed now.

File: integration_tests/jwt_aa_test.go [hunk 6]
```diff
@@ -328,6 +328,7 @@ func TestJWTAbstractAccount(t *testing.T) {
 	output, err = ExecBroadcast(t, ctx, xion.FullNodes[0], jsonTx)
 	require.NoError(t, err)
 	t.Logf("output: %s", output)
+	// Tx is failing
```
Comment (ash-burnt) [inline integration_tests/jwt_aa_test.go:331]: Is this valid?

File: integration_tests/minimum_fee_test.go [hunk 7]
```diff
@@ -35,6 +65,8 @@ func TestXionMinimumFeeDefault(t *testing.T) {
 		)
 		require.NoError(t, err)
 
+		fmt.Println("waiting check logs")
```
Comment (ash-burnt) [inline integration_tests/minimum_fee_test.go:68]: Probably not needed?

File: integration_tests/jwt_aa_test.go (hunk 6 above)
Comment (edjroz) [reply integration_tests/jwt_aa_test.go:331]: Nope.
```suggestion
```

File: integration_tests/minimum_fee_test.go (hunk 7 above)
Comment (edjroz) [reply integration_tests/minimum_fee_test.go:68]:
```suggestion

```

**Review by froch — APPROVED** [review]
👍

**Review by ash-burnt — APPROVED** [review]

**Pull request merged**